app/Http/Controllers/CollectionController.php

```diff
@@ -31,19 +32,63 @@
 use Illuminate\Http\JsonResponse;
 use Illuminate\Http\RedirectResponse;
 use Illuminate\Http\Request;
-use Illuminate\Pagination\LengthAwarePaginator;
+use Illuminate\Pagination\Paginator;
+use Illuminate\Support\Collection as SupportCollection;
 use Illuminate\Support\Facades\Cache;
 use Inertia\Inertia;
 use Inertia\Response;
+use Spatie\LaravelData\DataCollection;
 use Spatie\LaravelData\PaginatedDataCollection;
 
 class CollectionController extends Controller
 {
     public function index(Request $request): Response|JsonResponse|RedirectResponse
+    {
+        return Inertia::render('Collections/Index', [
+            'allowsGuests' => true,
+            'filters' => fn () => $this->getFilters($request),
+            'title' => fn () => trans('metatags.collections.title'),
+            'collectionsOfTheMonth' => fn () => $this->getCollectionsOfTheMonth(),
+            'collections' => fn () => $this->getPopularCollections($request),
+            'featuredCollections' => fn () => $this->getFeaturedCollections($request),
+            'votableCollections' => fn () => $this->getVotableCollections($request),
+        ]);
+    }
+
+    /**
+     * @return DataCollection<int, CollectionOfTheMonthData>
+     */
+    private function getCollectionsOfTheMonth(): DataCollection
+    {
+        return CollectionOfTheMonthData::collection(Collection::winnersOfThePreviousMonth()->limit(3)->get());
+    }
+
+    /**
+     * @return SupportCollection<int, VotableCollectionData>
+     */
+    private function getVotableCollections(Request $request): SupportCollection
+    {
+        $wallet = $request->wallet();
+
+        $userVoted = Collection::whereHas('votes', fn ($q) => $q->inCurrentMonth()->where('wallet_id', $wallet->id))->exists();
```

Comment:
We need to get the user voted collection in order to display it in the table: 

> If a user nominated (and voted for) another other collection that wasn’t in top 8, we show it at the last spot (replacing the actual 8th one) and on the left we’ll show the actual rank of it (e.g. 9 or higher)

Also we need its rank: ordered by votes and volume - this will enforce us to have `index` in the `VotableCollectionData`.

![image](https://github.com/ArdentHQ/dashbrd/assets/132887516/82d27bc3-2c3d-4fee-9eb4-e6e37299d623)

Reply:
@shahin-hq, I'm not sure if I'm going to finish this today so will leave the comment here just in case it affects something, since the voted item needs to be the eighth item on desktop and the [fourth item on mobile](https://www.figma.com/file/2Rr7zzoeGGiHYsROvfHaGe/Dashboard?type=design&node-id=26270-70877&mode=design&t=8zCVFFMp0fuT9BkF-4), I think the best option is to retrieve that item separately and merge it into the list on the frontend according to that logic.

Reply:
@shahin-hq updated according to that logic